ow_plexil/src/plans/TestFaultDependencies.plp

```diff
@@ -0,0 +1,43 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "plan-interface.h"
+
+// Tests the FaultDependencies class. Depending on the XML file used, 
+// the fault info of a subsystem will be printed out.
+
+
+TestFaultDependencies:
+{
+
+  PrintNode:
+  {
+    Repeat true;
+    Wait 3;
+
+    if !Lookup(IsOperable("Power")){
+      log_info("Power Subsystem Inoperable.");
+      log_info("Power Subsystem Faulty: ", Lookup(IsFaulty("Power")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Power")), "\n");
+    }
+
+    if !Lookup(IsOperable("Antenna")){
+      log_info("Antenna Subsystem Inoperable.");
+      log_info("Antenna Subsystem Faulty: ", Lookup(IsFaulty("Antenna")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Antenna")), "\n");
+    }
+
+    if !Lookup(IsOperable("Arm")){
+      log_info("Arm Subsystem Inoperable.");
+      log_info("Arm Subsystem Faulty: ", Lookup(IsFaulty("Arm")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Arm")), "\n");
+    }
+    if !Lookup(IsOperable("Camera")){
+      log_info("Camera Subsystem Inoperable.");
+      log_info("Camera Subsystem Faulty: ", Lookup(IsFaulty("Camera")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Camera")), "\n");
+    }
```

Comment:
I cannot get this text to print either when I tick on `camera_left_trigger_fault`. There is no warning text either. 

I believe the camera fault is only detected when a CameraCapture is attempted, so I called the action and still no output from this plan.

Reply:
I will do a clean build and try this again. Something must've been wrong on my end.

Reply:
Yes, that is correct! Thanks!